build.sh

```diff
@@ -255,23 +255,25 @@ build_gmake() {
     cp make gmake
 }
 
-JDK_DIR=openjdk-8u72b15-bsd-port-20160220
+# We use the JDK we just built for all consequent Java compilation (and as a
+# basis for the Graal compiler).
 case `uname` in
-    Linux)
-      JDK_JAVAC=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/jdk/bin/javac;;
-    OpenBSD)
-      JDK_JAVAC=${wrkdir}/openjdk/build/bsd-x86_64-normal-server-release/jdk/bin/javac;;
+    Linux)   OUR_JAVA_HOME=${wrkdir}/openjdk/build/linux-x86_64-normal-server-release/images/j2sdk-image/;;
```

Comment:
Uses of `OUR_JAVA_HOME`:

 * `if [ -f ${OUR_JAVA_HOME}/bin/javac ]; then return; fi` references a javac
 * `MX="env JAVA_HOME=${OUR_JAVA_HOME} python2.7 ${wrkdir}/mx/mx.py" `references a java jdk  image
 * `JAVACMD=${OUR_JAVA_HOME}/bin/java` references a java vm binary

All are in standard locations relative to a JAVA_HOME. So if anything, we want `OUR_JAVA_HOME`, no?